rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -514,6 +514,24 @@ void getPendingFederatorPublicKeyOfType() {
             assertThat(bridgeSupport.getPendingFederatorPublicKeyOfType(0, FederationMember.KeyType.MST), is(mstKey.getPubKey()));
         }
 
+        @Test
+        void getProposedFederation_whenProposedFederationExists_shouldReturnProposedFed() {
+            when(federationSupport.getProposedFederation()).thenReturn(Optional.of(federation));
+
+            Optional<Federation> proposedFed = bridgeSupport.getProposedFederation();
+            assertTrue(proposedFed.isPresent());
+            assertThat(proposedFed.get(), is(federation));
+        }
+
+        @Test
+        void getProposedFederation_whenBridgeSupportReturnsEmpty_shouldReturnEmpty() {
```

Comment:
```suggestion
        void getProposedFederation_whenFederationSupportReturnsEmpty_shouldReturnEmpty() {
```
Is that what you meant?

Reply:
updated